tests/test_estimators/test_lad_regression.py

```diff
@@ -59,7 +59,7 @@ def test_coefs_and_intercept__no_noise_regularization(coefs, intercept):
     """Test model with regularization. The size of the coef vector should shrink the larger alpha gets."""
     X, y = _create_dataset(coefs, intercept)
 
-    lads = [LADRegression(alpha=alpha, l1_ratio=0.5).fit(X, y) for alpha in range(3)]
+    lads = [LADRegression(alpha=alpha, l1_ratio=0.).fit(X, y) for alpha in range(3)]
```

Comment:
Is there a reason why the LADRegression test needed to be changed?

Reply:
I just noticed by accident. Dunno why I put a 0.5 there in the first place.

I just wanted to impose an L2 regularization, and that's what I check I also check afterward. in the next lines, I check if the sum of squares of coefficients goes down the larger alpha gets, and not a mix of the L1 and L2 norms of the coefficient.

The test still worked for the LADRegression, but it was just a happy little accident. For the QuantileRegression, the same check didn't pass, and that's when I noticed this mistake.